package.json

```diff
@@ -1,12 +1,8 @@
 {
   "scripts": {
-    "test": "ts-mocha --watch --watch-extensions ts -p tsconfig.json **/*.spec.ts",
-    "test-ci": "ts-mocha -p tsconfig.json **/*.spec.ts",
-    "dev": "dotenv -e .env.development next dev",
-    "build": "next build",
-    "start": "next start",
```

Comment:
Are we using `dev`, `build`, and `start`?

Reply:
i used dev before we set up our testing script. These 3 are out-of-the-box scripts for NextJS projects